### PR TITLE
[Community] Resolve releasing error for `llama-index-embeddings-ipex-llm` with Intel GPU supports

### PR DESCRIPTION
**Pull request by Oscilloscope98**
# Description

Resolve releasing error for `llama-index-embeddings-ipex-llm` with Intel GPU supports

Fixes https://github.com/run-llama/llama_index/actions/runs/8840374387/job/24275577585

This PR will cause no usage change. Small documentation update is also included in this PR to fit with latest `ipex-llm`.

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [x] No, as the previous version was not released successfully

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml
Comment (nerdai) [inline llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml:1]: please also bump the version number of the package.

**Review by nerdai — APPROVED** [review]
thanks @Oscilloscope98! Left a minor comment.

File: llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml
Comment (logan-markewich) [reply llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml:1]: actually v0.1.1 never published, and this should fix it 🙏🏻

Comment (nerdai) [reply llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml:1]: Ahh ok. 👍

**Pull request merged**